app/beacon-omop-worker/scripts/worker.py

```diff
@@ -2,20 +2,58 @@
 import sys
 import logging
 import beacon_omop_worker.config as config
+import json
+import argparse
 from beacon_omop_worker.db_manager import SyncDBManager
+from beacon_omop_worker import query_solvers
+
+parser = argparse.ArgumentParser(
+    prog="beacon-omop-worker",
+    description="This program executes beacon queries against an OMOP database",
+)
+subparsers = parser.add_subparsers(
+    dest="command", help="Commands to run", required=True
+)
+filters = subparsers.add_parser("filteringterms", help="Extract filtering terms")
+filters.set_defaults()
+
+
+def save_to_output(filters, destination) -> None:
```

Comment:
It's worth adding type annotations to everything, and use mypy early on to enforce it

Reply:
Yeah good idea, I added the missing type annotations for now and I'll have a look at mypy for future changes